exercises/sublist/SublistTest.cs

```diff
@@ -1,150 +1,109 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+// This file was auto-generated based on version 1.1.0 of the canonical data.
+
 using Xunit;
+using System.Collections.Generic;
 
 public class SublistTest
 {
     [Fact]
-    public void Empty_equals_empty()
+    public void Empty_lists()
     {
-        var list1 = new List<int>();
-        var list2 = new List<int>();
-        Assert.Equal(SublistType.Equal, Sublist.Classify(list1, list2));
+        Assert.Equal(SublistType.Equal, Sublist.Classify(new List<int>() { }, new List<int>() { }));
     }
 
     [Fact(Skip = "Remove to run test")]
-    public void Empty_is_a_sublist_of_anything()
+    public void Empty_list_within_non_empty_list()
     {
-        var list1 = new List<int>();
-        var list2 = new List<int> { 1, 2, 3, 4 };
-        Assert.Equal(SublistType.Sublist, Sublist.Classify(list1, list2));
+        Assert.Equal(SublistType.Sublist, Sublist.Classify(new List<int>() { }, new List<int>() {1, 2, 3}));
```

Comment:
Could you make it such that there is a space after the opening `{` and before the closing `}`? (also goes for the other tests).